pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/ExtensibleLoadManagerImpl.java

```diff
@@ -180,6 +184,24 @@ public class ExtensibleLoadManagerImpl implements ExtensibleLoadManager {
             .build();
     private final CountDownLatch initWaiter = new CountDownLatch(1);
 
+    /**
+     * Get all the bundles that are owned by this broker.
+     */
+    public Set<NamespaceBundle> getOwnedServiceUnits() {
+        var entrySet = serviceUnitStateChannel.getOwnershipEntrySet();
+        var brokerId = brokerRegistry.getBrokerId();
```

Comment:
"As I haven't come across this usage in the project so far"

This is not true. Pulsar already has been using "var".

Reply:
"As I haven't come across this usage in the project so far"

In fact, Pulsar already has been using "var".

Reply:
https://github.com/apache/pulsar/blob/master/pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/channel/ServiceUnitStateChannelImpl.java#L1141

You can see the `ServiceUnitStateChannelImpl` class.

Reply:
I agree, we are not using "var" yet.

Reply:
IMO, It's ok to use `var` in the new features. Since we are already in the JDK17 and `var` will help avoid very long code causes us to have to break code into multiple lines to follow `checkstyle` rule.
For existing code, we should avoid using any new API&Feature to avoid get compile error when cherry-pick patch to previous branches.